discord/user.py

```diff
@@ -110,8 +110,11 @@ def avatar_url_as(self, *, format, size=1024):
         format: Optional[str]
             The format to attempt to convert the avatar to.
             If the format is ``None``, then it is automatically
-            detected into either 'gif' or 'webp' depending on the
+            detected into either 'gif' or static_format depending on the
             avatar being animated or not.
+        static_format: Optional['str']
```

Comment:
This isn't `Optional` (this is reserved for parameters that take in `None` as a valid option).